config.js

```diff
@@ -0,0 +1,6 @@
+const host   = process.env.host || 'localhost';
+const port   = process.env.port || 1337;
+const config = { host, port };
+
+export { host, port, config };
+//export default config;
```

Comment:
let's remove all redundant comments (here and in [the index file](https://github.com/viktoria2506/node-server-stub/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bcR6))